src/Grace/DependencyInjection/Impl/ActivationStrategyCompiler.cs

```diff
@@ -283,6 +283,7 @@ public InjectionStrategyDelegate CreateInjectionDelegate(IInjectionScope scope,
         }
 
 
+        //TODO: Typo in method name.
```

Comment:
Just spotted this while searching the code for magic strings.
`CompileExpressionResultToOpitimzed` has a typo, but haven't touched it since it's a breaking change.